dashboard/src/modules/components/HeaderComponent/index.jsx

```diff
@@ -104,7 +108,7 @@ const HeaderToolbar = () => {
                 aria-label="Login"
                 className="header-login-button"
                 variant={ButtonVariant.plain}
-                onClick={() => navigate("/login")}
+                onClick={() => navigate("/dashboard/login")}
```

Comment:
Shouldn't this be referencing `APP_ROUTES.AUTH_LOGIN`?

Reply:
Yes, it should be